hummingbot/connector/derivative/binance_perpetual/binance_perpetual_utils.py

```diff
@@ -47,20 +48,6 @@ def split_trading_pair(trading_pair: str) -> Optional[Tuple[str, str]]:
         return None
```

Comment:
`split_trading_pair` above and the infrastructure around it (`SPECIAL_PAIRS`, `RE_4_LETTERS_QUOTE`, etc) can be removed now that we use the run-time generated map